cmake/unix.cmake

```diff
@@ -26,7 +26,11 @@ include_directories(${LIBUV_INCLUDE_DIRS})
 
 function(check_working_cxx_atomics64 varname)
   set(OLD_CMAKE_REQUIRED_FLAGS ${CMAKE_REQUIRED_FLAGS})
-  set(CMAKE_REQUIRED_FLAGS "${CMAKE_REQUIRED_FLAGS} -std=c++14")
+  if (EMBEDDED_CFG)
```

Comment:
Can remove the duplication + else block here by moving the -std=c++14 out of the if.

Reply:
what I meant was:

```cmake
set(CMAKE_REQUIRED_FLAGS "${CMAKE_REQUIRED_FLAGS} -std=c++14")
if (EMBEDDED_CFG)
  set(CMAKE_REQUIRED_FLAGS "${CMAKE_REQUIRED_FLAGS} -m32 -march=i486")
endif()
```